include/itkLabelSetMorphBaseImageFilter.hxx

```diff
@@ -154,7 +154,7 @@ void
 LabelSetMorphBaseImageFilter< TInputImage, doDilate, TOutputImage >
 ::GenerateData(void)
 {
-  ThreadIdType nbthreads = this->GetNumberOfThreads();
+  ThreadIdType nbthreads = this->GetNumberOfWorkUnits();
```

Comment:
I had noticed it at various points in ITK itself and the remotes as well. Since the name of the variable is not consistent across all of them, this would require a non-negligible amount of work. May be an issue in ITK would help having this in mind. If no objections exist, I'll open the issue tonight, and the desired variable naming can be discussed there.

Reply:
Good idea to address it in other places.

To avoid confusion, `workUnits` or `numberOfWorkUnits` will be more accurate than a variable that indicates it is the number of threads.

Reply:
I like `pieces`. It is short, and its use can be inferred from the method names it is used with.

Reply:
I do like `pieces` also: can remain unchanged when at some point in time multi-threading gets a new refactoring (not in the near future we hope :grin: ).

Reply:
`nbthreads` is not an English word, I may suggest something like `numberOfThreads` (already used in ITK), this would also help keep tool like codespell happy.

To get an idea, here are some stats from ITK master:

```
$ ack numberOfThreads | wc -l
179
$ ack nbthreads | wc -l
7
```